chronosphere/tfschema/resource_pools_config.go

```diff
@@ -124,3 +126,38 @@ var ResourcePoolPrioritiesSchema = &schema.Schema{
 	MaxItems: 1,
 	Optional: true,
 }
+
+var ResourcePoolAllocationThresholdsSchema = &schema.Schema{
+	Type: schema.TypeList,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"license": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"all_priorities":           ResourcePoolAllocationThresholdSchema,
+			"default_and_low_priority": ResourcePoolAllocationThresholdSchema,
+			"low_priority":             ResourcePoolAllocationThresholdSchema,
+		},
+	},
+	MinItems: 1,
+	Optional: true,
+}
+
+var ResourcePoolAllocationThresholdSchema = &schema.Schema{
+	Type: schema.TypeList,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"percent_of_pool_allocation": {
```

Comment:
both of these are optional, does one of them need to be set? If so you can use the terraform `ExactlyOneOf` method on `schema.Resource` to have terraform enforce that for you.